### ISSUE TITLE
Icon component throws error in CRA

### ISSUE THREAD

**Issue by MikaelColliander** [teambit/bit#4898]
### Description
Hi, any help would be deeply appreciated!

Icons work as expected on bit.dev:

https://bit.dev/sjse/component-library/ui/icons

When i install the component in CRA and try to render it, this warning is thrown:

`Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

This goes for any component containing icons.

### Specifications
* Bit version: 0.0.511
* Node version: 14.17.0
* npm / yarn version: 7.20.5 / 1.22.11
* Platform: OSX

### Context and additional information
I'm testing my components with Create React App:

In terminal `npm i @sjse/component-library.ui.icons`

Then in `App.tsx`:

```
import React from 'react';
import { IconList } from '@sjse/component-library.ui.icons';
import './App.css';

function App() {
  return (
    <div className="App">
      {React.version}
      <IconList />
    </div>
  );
}

export default App;
```

This produces a page with an error message:

![image](https://user-images.githubusercontent.com/2895442/134683287-86eb2311-c4cc-4aee-b836-becd532c7b34.png)

**Comment by KutnerUri** [comment]
couple of things to try:

1. locally getting this error:
```
Failed to compile
./src/App.tsx
Module not found: Can't resolve '@svgr/webpack' in '/Users/kutner/projects/temp/check-svg-components/src'
```
make sure `@svgr/webpack` is installed, and updated.

2. importing the component recursively from itself seems like a bad practice:
```tsx
import { iconList, Icon } from "@sjse/component-library.ui.icons";
```

I would change it to be:
```tsx
import { iconList } from "./iconList";
import { Icon } from "./icon";
```

The code is kinda hard to follow. How about something like this?
```
// icons.ts
export { ReactComponent as Cancel } from "@sjse/component-library.assets/Icons/24px/cancel_24px.svg";
export { ReactComponent as CancelFilled } from "@sjse/component-library.assets/Icons/24px/cancel_filled_24px.svg";
export { ReactComponent as Car } from "@sjse/component-library.assets/Icons/24px/car_24px.svg";
export { ReactComponent as Card } from "@sjse/component-library.assets/Icons/24px/card_24px.svg";
export { ReactComponent as Carrige } from "@sjse/component-library.assets/Icons/24px/carriage_24px.svg";

// iconList.ts
import * as icons from "./icons";

export const icons = Object.entries(icons).map((key, icon) => ({ [key]: icon }));
```

3. accessing individual files from other packages is a bad practice.  
  consider exporting the icons directly form the `assets` component.

**Comment by itaymendel** [comment]
closing issue as we got no reply from original author.
